tst/EnergyPlus/unit/DataHeatBalance.unit.cc

```diff
@@ -985,3 +985,50 @@ TEST_F(EnergyPlusFixture, DataHeatBalance_setNodeSourceAndUserTemp)
     EXPECT_EQ(expectedNodeNumberAtSource, thisConstruct.NodeSource);
     EXPECT_EQ(expectedNodeNumberAtUserSpecifiedLocation, thisConstruct.NodeUserTemp);
 }
+
+TEST_F(EnergyPlusFixture, DataHeatBalance_AssignReverseConstructionNumberTest)
+{
+    int ConstrNum;
+    int expectedResultRevConstrNum;
+    int functionResultRevConstrNum;
+    bool ErrorsFound = false;
```

Comment:
Don't forget, in C++, you don't have to declare the variables up here at the top of the function, just declare them at the point you use them.  :+1: